### ISSUE TITLE
[Question] Is the project dead? Is so, what are options to migrate to?

### ISSUE THREAD

**Issue by jadbox** [cerebral/overmind#552]
As just an outside observer, there hasn't been any updates for many months now despite their being open PRs and unhandled bugs in the issue tracker. If the project is indeed dead (plz correct me if I'm wrong), what have people move to as a good alternative?

**Comment by NotoriousPyro** [comment]
I'm going to take a look at MobX instead. This project was great when I found it, but unfortunately un-maintained https://gist.github.com/christianalfoni/f1c4bfe320dcb24c403635d9bca3fa40

**Comment by christianalfoni** [comment]
Hi there!

Thanks for writing the issue! Please join our Discord server and you will get some more information about the state of the project.

In short the latest version is the stable API and any bugs that appears will be handled. I have tried to do this with transferring maintenance responsibility, but it is a lot of work to stay on top of issues and fix them at the same time. I am trying a new strategy now where I am still involved, but keep the predictability needed to make it all work with my day job and family 😄 

Please join the Discord to see what is happening there

**Comment by lavir** [comment]
@christianalfoni, I'm super happy that you stand with us, users, and try to do everything to keep the project alive.
We use Overmind a lot, the project is huge, and even migrating from v27 to v28 is a problem, I'm not talking about migrating to another alternative. Nevertheless, just in case I did try last moths others alternatives such as Mobx, StateJS, Jotai, and Hookstate. I used Redux in the past, not happy at all, especially when I found Overmind :)

I would say that Overmind is the best of all SMs. Really. I will be very happy if you continue to keep the project. 

Thank you, @christianalfoni !
